### DISCUSSION TITLE
Blazor web server: Filter behavior with new grid version

### DISCUSSION THREAD

**Discussion by Malori331977** [vikramlearning/blazorbootstrap#878] [General]
I have noticed a strange behavior when typing something in the grid filters, the letters are erased or change if one types too fast.  I went to check out the demo that is on the website and it behaves the same.  My application is server-side. 
Do you know what it can be?

**Comment by gvreddy04** [comment]
@Malori331977 Thank you for using **BlazorBootstrap**. Please share a screen recording that demonstrates your scenario on our demo pages. Additionally, please confirm your network speed.

**Comment by Malori331977** [comment]
[Blazor Grid Component - Filters.zip](https://github.com/user-attachments/files/17099394/Blazor.Grid.Component.-.Filters.zip)

Hello, thank you very much for replying.  In that video I'm trying to filter the grid data using the names Alice and Ronald.  If I type slowly the grid works fine, but if I type fast the names in the filter sometimes it does not behave correctly.
My internet speed is 200mb on average.

**Reply by OriginalSkain** [reply]
[Bootstrap Demo - Grid Filters](https://github.com/user-attachments/assets/1ae28942-475a-4141-b965-c0601b46895a)
You can see this issue on the existing demo site. I’m currently investigating it in order to raise a fix, but I haven’t been able to pinpoint the root cause yet. This issue also affects the Autocomplete control.

**Reply by OriginalSkain** [reply]
Added PR to fix this issue #1248
